### PR TITLE
Implementation Description and Testimonials from Siemens and Bosch

### PR DESCRIPTION
**Pull request by egekorkan**
- I have added testimonials from Siemens and Bosch
- I have added implementation descriptions of node-wot, sayWoT, ditto, editdor and playground
- I have quickly formatted hitachi implementation description

### CODE REVIEW

**Comment by egekorkan** [comment]
Additionally, @sebastiankb, @thjaeckle and @fexpal can you have a look? I cannot add you as reviewers

**Review by k-toumura — APPROVED** [review]

File: events/2022.03.Online/Architecture/Impls/ditto.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="impl" id="impl-ditto">
+  <h4 id="impl-ditto-WoT">Eclipse Ditto: TD Exposer</h4>
+  <p>
+    A service that exposes Eclipse Ditto managed digital twins with Thing
+    Descriptions. More information can be found at
+    <https://www.eclipse.org/ditto/basic-wot-integration.html>
+  </p>
+  <p><b>Usage:</b> Service</p>
+  <p><b>Public Repository:</b> https://github.com/eclipse/ditto</p>
+  <p><b>Contributing Member Organizations:</b> Bosch.IO</p>
```
Comment (thjaeckle) [inline events/2022.03.Online/Architecture/Impls/ditto.html:10]:
```suggestion
  <p>
    A service that provides APIs for managing digital twins and interacting with the 
    abstracted devices. Fetches linked Thing Models of digital twins and provides instance 
    specific Thing Descriptions for them.
  </p>
  <p><b>Usage:</b> Service</p>
  <p><b>Public Repository:</b> https://github.com/eclipse/ditto</p>
  <p><b>Website:</b> https://www.eclipse.org/ditto/basic-wot-integration.html</p>
  <p><b>Contributing Member Organizations:</b> Bosch.IO</p>
```

I would suggest the above instead - would that be ok?

File: events/2022.03.Online/Architecture/Impls/ditto.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  <h4 id="impl-ditto-wot-model">Eclipse Ditto WoT Model: Tool</h4>
+  <p>
+    Eclipse Ditto offers a Java Module to use TDs and TMs inside Java
+    environments with concepts native to Java.
```
Comment (thjaeckle) [inline events/2022.03.Online/Architecture/Impls/ditto.html:15]:
```suggestion
    Eclipse Ditto offers a Java Module to use TDs and TMs inside Java
    environments with concepts native to Java, e.g. providing a builder API for creating 
    new TM/TD instances and rendering those to JSON strings and vice versa.
```

**Review by thjaeckle — CHANGES_REQUESTED** [review]
I left some suggestions mostly regarding the Ditto implementation sections.

File: events/2022.03.Online/Architecture/Testimonials/siemens.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      block for several digitalization topics such as device onboarding, device
+      management, digital twins, and data analytics.
+    </p>
+    <p>
+      Siemens commits to several open-source implementations of W3C Web of
+      Things, which are managed in the public Eclipse Thingweb and Eclipse
+      EdiTDor projects.
+    </p>
+    <p>
+      Siemens also implements WoT in its sayWoT! offering that can be used IoT
```
Comment (felpal) [inline events/2022.03.Online/Architecture/Testimonials/siemens.html:19]:
```suggestion
      Siemens also implements WoT in its <a href="https://www.evosoft.com/digitalisierungsangebot/saywot/">sayWoT!</a> offering that can be used in IoT
```

Would it be possible to add this?

**Comment by egekorkan** [comment]
@thjaeckle and @felpal I have incorporated your feedback

**Review by thjaeckle — APPROVED** [review]

**Pull request merged**